src/yaml-support/schema-holder.ts

```diff
@@ -37,14 +40,49 @@ export class KubernetesClusterSchemaHolder {
 
     private async loadSchemaFromActiveCluster(kubectl: Kubectl, schemaEnumFile?: string): Promise<void> {
         const clusterSwagger = await swagger.getClusterSwagger(kubectl);
-        const crdSchemas = await swagger.getCrdSchemas(kubectl);
-        if (crdSchemas) {
-            this.crdSchemas = crdSchemas;
+        const fetchCRDSchemasEnabled = await this.isFetchingCRDsAllowed(kubectl);
+        if (!fetchCRDSchemasEnabled) {
```

Comment:
I'm wondering if this should not have the negation.

Reply:
Yes, you're right

Reply:
Thanks!!